src/pages/spawn/spawnVolume/SpawnVolumeTable.tsx

```diff
@@ -66,7 +65,6 @@ const columns: Array<ColumnProps<MyVolume>> = [
         <WordBreak>{getHostDisplayName(volume)}</WordBreak>
       </Link>
     ),
-    width: 400,
```

Comment:
Given that the "Action" column is pretty wide now, setting a fixed width here made the columns too likely to overflow or break awkwardly.